test/unit/project_changed_state.js

```diff
@@ -48,7 +48,11 @@ test('Adding a sprite (from sprite3) should emit a project changed event', t =>
     vm.addSprite(sprite3).then(() => {
         t.equal(projectChanged, true);
         t.end();
-    });
+    })
+        .catch(e => {
```

Comment:
Why is this change necessary?

Reply:
Wrote this last week. But its posting was stucking a review queue. I hit the wrong button.

Reply:
Gonna remove this catch statement.

Reply:
LG! Thanks!